Python/Product/Uwp/Interpreter/PythonUwpInterpreterFactoryProvider.cs

```diff
@@ -23,39 +25,68 @@ namespace Microsoft.PythonTools.Uwp.Interpreter {
     class PythonUwpInterpreterFactoryProvider : IPythonInterpreterFactoryProvider {
         private HashSet<IPythonInterpreterFactory> _factories = null;
 
-        public const string DefaultVersion = "3.5";
-
         public event EventHandler InterpreterFactoriesChanged;
 
         public PythonUwpInterpreterFactoryProvider() {
-            PythonVersion = DefaultVersion;
-        }
-
-        public string PythonVersion {
-            get; set;
         }
 
         private void DiscoverFactories() {
             if (_factories == null) {
-                var defaultConfig = new InterpreterConfiguration(
-                    null,
-                    null,
-                    null,
-                    null,
-                    null,
-                    ProcessorArchitecture.None,
-                    new Version(PythonVersion),
-                    InterpreterUIMode.CannotBeDefault
-                );
+                var userSdkInstallDir = new DirectoryInfo(Environment.ExpandEnvironmentVariables(@"%LOCALAPPDATA%\Microsoft\VisualStudio\%VISUALSTUDIOVERSION%Exp\Extensions\Microsoft\Python UWP"));
+                var sdkInstallDir = new DirectoryInfo(Environment.ExpandEnvironmentVariables(@"%ProgramFiles(x86)%\Microsoft SDKs\Windows\v10.0\ExtensionSDKs\Python UWP"));
 
                 _factories = new HashSet<IPythonInterpreterFactory>();
 
-                _factories.Add(new PythonUwpInterpreterFactory(defaultConfig));
-
-                var factoriesChanged = InterpreterFactoriesChanged;
-
-                if (factoriesChanged != null) {
-                    factoriesChanged(this, new EventArgs());
+                if (userSdkInstallDir.Exists) {
+                    foreach (var dirInfo in userSdkInstallDir.EnumerateDirectories()) {

```

Comment:
It'll be safer to have this block in its own function (since it's duplicated) and protected with a try/catch. All sorts of random errors can happen when dealing with the file system, and if this code is running off the UI thread (likely) then an error will bring down VS completely